SPECS/at/at.spec

```diff
@@ -128,13 +128,13 @@ make test
 touch %{_localstatedir}/spool/at/.SEQ
 chmod 600 %{_localstatedir}/spool/at/.SEQ
 chown root:root %{_localstatedir}/spool/at/.SEQ
-%{systemd_post} atd.service
```

Comment:
Shouldnt these effectively be the same thing? For my own understanding, when do we wrap in {}?